astropy/time/core.py

```diff
@@ -1971,17 +1973,17 @@ def _get_delta_tdb_tt(self, jd1=None, jd2=None):
             ut = day_frac(njd1 - 0.5, njd2)[1]
 
             if self.location is None:
-                from astropy.coordinates import EarthLocation
-                location = EarthLocation.from_geodetic(0., 0., 0.)
+                # Assume geocentric.
+                self._delta_tdb_tt = erfa.dtdb(jd1, jd2, ut, 0., 0., 0.)
```

Comment:
I assume this is done instead of `location = EarthLocation.from_geocentric(0., 0., 0.)` to short-circuit the calculation and avoid the extra operations?

Reply:
Yes, indeed. I probably should note that...

Reply:
Yes, I like this simplification